projects/aca-content/src/lib/components/info-drawer/metadata-tab/metadata-tab.component.ts

```diff
@@ -111,6 +105,6 @@ export class MetadataTabComponent implements OnInit, OnDestroy {
   }
 
   private checkIfNodeIsUpdatable(node: Node) {
-    this.canUpdateNode = node && !isLocked({ entry: node }) ? this.permission.check(node, ['update']) : false;
+    this.readOnly = !(node && !isLocked({ entry: node }) ? this.permission.check(node, ['update']) : false);
```

Comment:
should be moved to metadata component where it checks for allowable operations

Reply:
Denys has changed this code. Not getting where should we move this exactly.